bitshuffle/bitshuffle.py

```diff
@@ -361,16 +365,30 @@ def infer_mode(args):
 
 def set_defaults(args):
 
-    defaults = {'input': '/dev/stdin', 'output': '/dev/stdout',
+    defaults = {'input': stdin, 'output': stdout,
```

Comment:
Well done! Didn't even consider that we could do this.

Reply:
This is one of my favorite python-isms. It's also nice because it's entirely out-of-band from the text console.